docs/content/preview/launch-and-manage/monitor-and-alert/metrics/cache-storage.md

```diff
@@ -106,3 +108,13 @@ The Write Ahead Log (or WAL) is used to write and persist updates to disk on eac
 | `log_reader_bytes_read` | bytes | counter | Number of bytes read from WAL after the tablet start. |
 
 These metrics are available per tablet and can be aggregated across the entire cluster using appropriate aggregations.
+
+# YSQL cache metrics
+
+## Catalog cache misses
+
+During YSQL query processing, system catalog (pg_catalog) tables that live on the yb-master are cached locally on each YSQL backend process. Misses on this cache can make initial queries or queries after a DDL change slow until the corresponding cache is warmed up. The following table describes metrics for the specific pg_catalog tables which were not found in the cache and required a yb-master lookup. Such tables can then be preloaded by using [a tserver gflag](../../../reference/configuration/yb-tserver/#ysql-catalog-preload-additional-table-list).
+
+| Metric | Unit | Type | Description |
+| :------ | :--- | :--- | :---------- |
+| `handler_latency_yb_ysqlserver_SQLProcessor_CatalogCacheTableMisses_count` | misses | counter | Count of catalog cache misses for this pg_catalog table or an associated index  |
```

Comment:
Really too long for a table, might have to do this another way, just a bullet or something

Reply:
Let me know. Yes it is unreadable on a narrow screen, but not sure how to fix it